src/rebar_prv_xref.erl

```diff
@@ -36,7 +36,7 @@ init(State) ->
 
 -spec do(rebar_state:t()) -> {ok, rebar_state:t()} | {error, string()}.
 do(State) ->
-    rebar_paths:set_paths([deps], State),
+    rebar_paths:set_paths([runtime], State),
```

Comment:
@ferd IIRC you were the one who reworked how rebar3 handles the paths, so if this change should instead be `[deps, runtime]` or anything else, let me know :)

Reply:
The `[deps, runtime]` approach would basically say "load all the deps, then put the runtime deps after them in the path". We use that form for `[deps, plugins]` or `[plugins, deps]` when we need both to be present, but favor one type of dependency _if_ some of the apps in there clash (many versions exist).

Since the objective of this PR is to run _only_ on runtime deps, then I'd expect the current `[runtime]` to be correct.